admin-dev/themes/default/template/controllers/categories/helpers/form/form.tpl

```diff
@@ -41,6 +41,13 @@
 	{else}
 		{$smarty.block.parent}
 	{/if}
+	{if ($input.name == 'active')}
+		<div class="col-lg-12">
+			<div class="help-block">
+				{l s='If you want a category to appear in the menu of your shop, go to Modules > Modules & Services > Installed modules. Then, configure your menu module.' d='Admin.Catalog.Help'}
```

Comment:
We can improve the message by adding a link to the Module page if you want to. :)

Reply:
Why not. @LouiseBonnard?

Reply:
Okay for both, is it possible to have the link opened in a new tab (in order to avoid the loss of the modifications already done on this page) ?